client/src/app/s-degree-plan/s-degree-plan.component.html

```diff
@@ -12,7 +12,7 @@
       class="flex-column justify-content-center w100"
     >
       <!-- Year -->
-      <div *ngFor="let year of plan" class="year">
+      <div *ngFor="let year of plan.years" class="year">
```

Comment:
Since I added a years attribute to the JSON object returned from planService.formatPlan(), I need to call plan.years to access the years array